src/Calculator/SettingsPage.xaml

```diff
@@ -0,0 +1,164 @@
+            <Grid.RowDefinitions>
+                <RowDefinition Height="1*"
+                           MinHeight="{StaticResource HamburgerHeight}"
+                           MaxHeight="52"/>
+                <RowDefinition Height="1*"/>
+            </Grid.RowDefinitions>
+
+            <Grid Height="{StaticResource HamburgerHeight}"
+              HorizontalAlignment="Left"
+              VerticalAlignment="Top">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="{StaticResource HamburgerHeightGridLength}"/>
+                    <ColumnDefinition Width="Auto"/>
+                </Grid.ColumnDefinitions>
+
+                <Button x:Name="BackButton"
+                    x:Uid="BackButton"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    Background="{ThemeResource AppChromeAcrylicHostBackdropMediumLowBrush}"
+                    FontFamily="Segoe MDL2 Assets"
+                    FontSize="15"
+                    Windows10version1809:CornerRadius="0"
+                    Click="BackButtonClick"
+                    Content="&#xE72B;"/>
+
+                <TextBlock x:Uid="SettingsHeader"
+                       Grid.Column="1"
+                       Margin="8,-3,0,0"
+                       HorizontalAlignment="Left"
+                       VerticalAlignment="Center"
+                       Style="{StaticResource CategoryNameTextBlockStyle}"/>
+            </Grid>
+
+            <StackPanel Grid.Row="1" Margin="12,0,12,0">
+                <StackPanel x:Name="SettingsStack"
+                        AutomationProperties.LabeledBy="{Binding ElementName=SettingsAppTheme}"
+                        Orientation="Vertical">
+
+                    <TextBlock x:Name="SettingsAppTheme"
+                           x:Uid="SettingsAppTheme"
+                           Style="{StaticResource SubtitleTextBlockStyle}"
+                           AutomationProperties.HeadingLevel="Level1"
+                           TextTrimming="Clip"/>
+
+                    <RadioButton x:Name="SettingsLightTheme"
+                             x:Uid="SettingsLightTheme"
+                             Grid.Row="3"
+                             Checked="LightChecked"/>
+
+                    <RadioButton x:Name="SettingsDarkTheme"
+                             x:Uid="SettingsDarkTheme"
+                             Grid.Row="2"
+                             Checked="DarkChecked"/>
+
+                    <RadioButton x:Name="SettingsSystemTheme"
+                             x:Uid="SettingsSystemTheme"
+                             Grid.Row="4"
+                             Checked="SystemChecked"/>
+
+
+                    <TextBlock x:Name="SettingsRestartApp"
+                           Style="{StaticResource CaptionTextBlockStyle}"
+                           Foreground="{StaticResource SystemColorGrayTextColor}"/>
```

Comment:
In dark mode the color of this text is hard to read
![image](https://user-images.githubusercontent.com/6334170/108274373-a7569800-7129-11eb-8336-33bda92b0eb4.png)